### PR TITLE
Add duels

### PR DESCRIPTION
**Pull request by KevinBasener**
Continuation for #48.

### CODE REVIEW

File: duel/duell_movement/duell.gd [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	var question_result = await question_panel.display_specific_question(current_question)
+	await get_tree().create_timer(1.5).timeout
+	remove_child(question_panel)
+	
+	return 1 if question_result else 0
+
+
+func _on_duel_result(ending):
+	match ending:
+		DuelEndings.ATTACKER_WINS:
```
Comment (hfxbse) [inline duel/duell_movement/duell.gd]: Looks like a debug placeholder, is this as intended?

File: main/main.gd [hunk 2]
```diff
@@ -82,6 +82,16 @@ func create_players(num):
...
 		player.sprite.visibility_layer = 9
 		
+		var regexPattern = "\\b(?!res\\b)\\w+\\b"
+		var regex = RegEx.new()
+		regex.compile(regexPattern)
+		var matches = regex.search_all(sprites[i].resource_path)
+		
+		player.player_name = matches[1].get_string()
+		
+	var duel = preload("res://duel/duell_movement/overtaking_duell.tscn").instantiate()
```
Comment (ubeF) [inline main/main.gd]: Whats the purpose of this?

File: duel/duell_movement/duell.gd (hunk 1 above)
Comment (KevinBasener) [reply duel/duell_movement/duell.gd]: Oh yes was for debugging

File: main/main.gd (hunk 2 above)
Comment (KevinBasener) [reply main/main.gd]: Maybe testing? Not sure anymore, but I think it can be removed.

**Review by hfxbse — DISMISSED** [review]
Looks passable once the left over development code is removed.

**Review by ubeF — APPROVED** [review]

**Pull request merged**